### PR TITLE
[READY TO MERGE] Update contributors.md with Brigitta and Kalle's Info

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Related to #3322 

* Add new bios (like for @kallewesterling from a recent Book Dash)
* Update bios where needed (like for @bsipocz)

### List of changes proposed in this PR (pull-request)

* Update contributors.md file

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Review by kallewesterling — APPROVED** [review]
The addition of my stuff looks good, thanks!

File: contributors.md [hunk 1]
```diff
@@ -418,23 +418,23 @@ I also hope that it can convince people who don't consider themselves capable of
...
+> I collaborated with a group of researchers from The Alan Turing Institute to draft a chapter of scientific outreach.
+
 ### Brigitta Sipőcz
 
-* Twitter: [@BenjaminMummery](https://twitter.com/@BenjaminMummery)
+* Role: Infrastructure Maintainer (2023 - present)
+* GitHub id: [bsipocz](https://github.com/bsipocz)
+* ORCID: [0000-0002-3713-6337](https://orcid.org/0000-0002-3713-6337)
+
 * Short bio:
```
Comment (aleesteele) [inline contributors.md:435]: @bsipocz - is there anything you would like to add here? 😊

File: contributors.md [hunk 2]
```diff
@@ -891,6 +891,30 @@ I am also an associate member of the Chartered Institute of Personnel and Develo
...
+* Personal highlights:
+> I am excited to become more involved with The Turing Way community after the inspiring experience of this Book Dash.
+> During the event, I learned a tremendous amount about JupyterBook. Learning more around the syntax for referencing, adding images, and captions was particularly good for me :) 
+> This experience has definitely empowered me to dive into other open-source projects, explore more hackathons, and engage in similar collaborative efforts across the open-source community.
+> I also learned more about how to manage the all-contributors bot... and when to give up and just re-start completely!
+> I reviewed more pull requests than ever before in such a short span — super fun!
+> My role also included a lot of cheering on from the sidelines, a supportive effort to encourage the closure of more pull requests... I love to see a closed PR!!
+> Using a Miro board to restructure an entire section of The Turing Way was a big highlight too. It provided an opportunity to collaborate hands-on with colleagues I hadn't worked with closely before. It was not only productive but also brought, for me, a sense of teamwork and community.
+
+* Personal Quote:
```
Comment (aleesteele) [inline contributors.md:914]: What a beautiful testimony, @kallewesterling!

**Comment by kallewesterling** [comment]
Is there a lazy consensus in place for these pull requests, @malvikasharan?

Is there anything you'd like to add/modify here, @bsipocz?

**Comment by KirstieJane** [comment]
I'd prefer not to follow a lazy consensus for details about people. So hopefully we can ping @bsipocz in the new year (if she doesn't see this before then!) and meet then 😄 

Thank you for the tidying @kallewesterling!! Such a gift 💝

File: contributors.md (hunk 1 above)
Comment (bsipocz) [reply contributors.md:435]: I need to rework and reword this bio. Once I get to it, will try to remember to update it here, too.

File: contributors.md [hunk 3]
```diff
@@ -418,23 +418,23 @@ I also hope that it can convince people who don't consider themselves capable of
...
+
 ### Brigitta Sipőcz
 
-* Twitter: [@BenjaminMummery](https://twitter.com/@BenjaminMummery)
+* Role: Infrastructure Maintainer (2023 - present)
+* GitHub id: [bsipocz](https://github.com/bsipocz)
+* ORCID: [0000-0002-3713-6337](https://orcid.org/0000-0002-3713-6337)
+
 * Short bio:
 > I am an astronomer turned Research Software Engineer. I work at Caltech/IPAC to build and improve tools, e.g. Python libraries and Science Platforms to provide ways to access data in the NASA/IPAC Infrared Science Archive. Prior joining IPAC I was DiRAC Fellow in the data engineering team at the Institute for Data Intensive Research in Astrophysics and Cosmology in Seattle. I am a developer and maintainer of several open source astronomy libraries and their infrastructure (e.g. astroML, astroquery, astropy) and I enjoy contributing to upstream projects as well. I have a keen interest in finding ways to promote the open development model and make tools more sustainable. I am a fellow of the Software Sustainability Institute.
```
Comment (bsipocz) [inline contributors.md:437]: GH didn't let me add a suggestion on the deleted lines, so I added it again. Feel free to edit/fix grammar/etc as needed.

```suggestion

* Personal highlights:
> Having the opportunity to work with graphic designers during the Book Dash events. It was fascinating to witness the iterations and evolution from (occasionally) fuzzy ideas into gorgeous illustrations. I keep using the material outside of TTW for my own talks about open source and collaboration.  
> The small zoom celebration we had with the infrastructure WG team as we migrated to The Turing Way github organization. 
```

**Review by bsipocz — APPROVED** [review]
Sorry @kallewesterling, this slipped through.

**Comment by kallewesterling** [comment]
@malvikasharan looks like this is ready to merge now?

**Comment by aleesteele** [comment]
Merged and closed! 🦾 🚀

**Comment by bsipocz** [comment]
This will need a follow-up to add my changes.

**Pull request merged**